src/components/posts/PostList.js

```diff
@@ -15,17 +18,69 @@ export const PostList = props => {
         getPosts()
     }, [])
 
+    const history = useHistory()
+
+    const handleDelete = postId => {
```

Comment:
Side note here: we could make `deletePost` automatically get the posts, and then we wouldn't need to define `handleDelete` in any function that has the confirmable delete button.

Reply:
As i just said with my human voice, we have different behavior in different contexts as far as deleting a post goes... so like if we deletePost in the context of "My Posts", we don't want to `getPosts` after that's done, we want to `getPostsByUserId`. 

THAT BEING SAID, this may impact a bit about the `ImagePostListItem` since it is currently written under the assumption that it will be getting posts by user id after deletion. We may have to defer `onDelete` to the parent of a `ImagePostListItem` instead of handling delete directly in that component because of this.

Reply:
🤣🤣🤣
That's a good point about the ImagePostListItem, lesson learned: Don't modularize too early!